app/server/api/metrics_api.py

```diff
@@ -130,6 +130,24 @@ def get(self):
 
         return make_response(jsonify(response_object)), 200
 
+class CacheApi(MethodView):
+    @requires_auth(allowed_roles={'ADMIN': 'any'})
```

Comment:
feels like this should be restricted to sempoadmin

Reply:
Done